apps/api/src/app/webhooks/webhooks.controller.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  Body,
+  ClassSerializerInterceptor,
+  Controller,
+  NotFoundException,
+  Param,
+  Post,
+  UseInterceptors,
+} from '@nestjs/common';
+import { IntegrationEntity, MessageRepository } from '@novu/dal';
+import { ChannelTypeEnum, EmailProviderIdEnum } from '@novu/shared';
+import { MailFactory } from '../events/services/mail-service/mail.factory';
+import {
+  GetDecryptedIntegrations,
+  GetDecryptedIntegrationsCommand,
+} from '../integrations/usecases/get-decrypted-integrations';
+
+@Controller('/webhooks')
+@UseInterceptors(ClassSerializerInterceptor)
+export class WebhooksController {
+  private mailFactory = new MailFactory();
+
+  constructor(
+    private getDecryptedIntegrationsUsecase: GetDecryptedIntegrations,
+    private messageRepository: MessageRepository
+  ) {}
+
+  @Post('/:orgId/:envId/email/:providerId')
```

Comment:
Hmmm I am not sure about this. But I at least would use the full names (environmentId etc)

Reply:
@davidsoderberg is it a post request for most of them right?

Reply:
Yes it seems to be standard 😄